WordPress/WordPressTest/JetpackCapabilitiesServiceTests.swift

```diff
@@ -23,8 +23,8 @@ class JetpackCapabilitiesServiceTests: CoreDataTestCase {
         let service = JetpackCapabilitiesService(coreDataStack: contextManager, capabilitiesServiceRemote: remoteMock)
 
         service.sync(blogs: [RemoteBlog.mock()], success: { blogs in
-            XCTAssertTrue(blogs.first!.capabilities["backup"] as! Bool)
-            XCTAssertTrue(blogs.first!.capabilities["scan"] as! Bool)
+            XCTAssertTrue(blogs.first!.capabilities["backup"]!)
+            XCTAssertTrue(blogs.first!.capabilities["scan"]!)
```

Comment:
`XCTAssertTrue` already expects a `Bool` as input, casting is (should be?) unnecessary. The compiler will (should?) figure it out.

Reply:
Correct, `public var capabilities: [String: Bool]` so no casting is needed.